Write "looking ahead" sections for all chapters (#511)
worldbank/dime-data-handbook: chapters/3-measurement.tex

or if results seem to depend dramatically on the outcomes for a small number of individuals,
randomization inference will flag those issues before the experiment is fielded
and allow adjustments to the design to be made.

\section{Looking ahead}

> This chapter provided a framework for you

Reviewer: ```suggestion
This chapter provided a framework for you to
```

> develop a data map that documents the design of your study.

Reviewer: ```suggestion
develop a data map that documents your data needs and 
how they relate to the design of your study.
```
Something like this is needed. it is not the design that it documents...

Author: Hmm, I don't agree, I think the point of master datasets and research variables _is_ to document the study design! or at least its execution.

Reviewer 2: Softly in favor of Kris's suggestion, i do think of these as providing a framework for linking all the data to the research questions

Author: Ok -- edited and adopted

or if results seem to depend dramatically on the outcomes for a small number of individuals,
randomization inference will flag those issues before the experiment is fielded
and allow adjustments to the design to be made.

\section{Looking ahead}
This chapter provided a framework for you

> plan the data work that corresponds to the design of your experiment.

Author: In this one I want to set up the "research variables" reference explicitly (in the next comment), to show that this is data work using the design information (the data map). Is there a clearer way to express that?

> plan the data work that corresponds to the design of your experiment.

Reviewer 2: i think that can be addressed in the next sentence, the phrasing here is so clunky ...

> plan the data work that corresponds to the design of your experiment.

Author: ```suggestion
develop a data map that documents the design of your study.
```

These research variables and the data map you create

> to catalog, link, and describe them

Author: In this one I want to leave the "research variables" reference explicitly, to show that this is data work _using_ the design information (the data map). Is there a clearer way to express that?

> to catalog, link, and describe them

Reviewer 2: Ok. the way this is phrased makes it seem like the data map is cataloging/linking/describing the research variables, that's what i didn't like. not very sure how to clarify. Maybe something like: You learned how to translate the research design into specific research variables, and created a data map to guide construction of the analysis dataset.

> These research variables and the data map you create
> to catalog, link, and describe them

Author: ```suggestion
The data map contains research variables and the instructions
for using them in combination with measurement variables, which together
```

> will form the foundation of your analytical work
> when you obtain measurements from your data.

Author: ```suggestion
will form the dataset for your analytical work.
```
